### PR TITLE
automate nav for GEPs

### PR DESCRIPTION
**Pull request by hbagdi**
**What type of PR is this?**

/kind cleanup
**What this PR does / why we need it**:
Automates the nav for GEPs
**Which issue(s) this PR fixes**:

None

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by hbagdi** [comment]
This works locally but Netlify blew up. can't access logs in Netlify at the moment(they might have an outage).

File: requirements.txt [hunk 1]
```diff
@@ -15,3 +15,4 @@ PyYAML==5.3
 six==1.14.0
 tornado==6.0.3
 mkdocs-macros-plugin==0.4.18
+mkdocs-awesome-pages-plugin=2.5.0
```
Comment (jpeach) [inline requirements.txt:18]:
```$ make .venv
Creating a virtualenv in .venv... 
Installing packages in .venv... 
ERROR: Invalid requirement: 'mkdocs-awesome-pages-plugin=2.5.0' (from line 18 of requirements.txt)
Hint: = is not a valid operator. Did you mean == ?
make: *** [Makefile:107: .venv] Error 1
```

**Review by jpeach — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (hbagdi) [reply requirements.txt]: Time to go for an eye check up.

Fixed.

**Comment by hbagdi** [comment]
Here is the preview: https://deploy-preview-744--kubernetes-sigs-gateway-api.netlify.app/

**Review by jpeach — APPROVED** [review]
Nice!

**Pull request merged**